templates/clockwork.toml.j2

```diff
@@ -62,8 +62,10 @@ display_order={{ cluster.display_order | default(9999) }}
 {% if cluster.sacct_enabled %}
 remote_user="{{ cluster.remote_user }}"
 remote_hostname="{{ cluster.remote_hostname }}"
+ssh_port={{ cluster.ssh_port }}
 sacct_enabled=true
 sacct_path="{{ cluster.sacct_path }}"
-sacct_ssh_key_filename="id_clockwork"
+sinfo_path="{{ cluster.sinfo_path }}"
+ssh_key_filename="id_clockwork"
```

Comment:
This should either be `"{{ cluster.ssh_key_filename | default('id_clockwork') }}" or not documented as configurable above.